packages/kbn-router-to-openapispec/src/process_versioned_router.ts

```diff
@@ -98,6 +98,7 @@ export const processVersionedRouter = (
         tags: route.options.options?.tags ? extractTags(route.options.options.tags) : [],
         ...(route.options.description ? { description: route.options.description } : {}),
         ...(route.options.deprecated ? { deprecated: route.options.deprecated } : {}),
+        ...(route.options.discontinued ? { discontinued: route.options.discontinued } : {}),
```

Comment:
Depending on the output, we might need to modify the way the "discontinued" value displays.
for example:
```
...(route.options.discontinued ? { discontinued: `discontinued from ${route.options.discontinued}` } : {})